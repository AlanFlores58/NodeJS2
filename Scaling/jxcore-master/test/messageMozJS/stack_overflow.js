// Copyright & License details are available under JXCORE_LICENSE file





var common = require('../common');
var assert = require('assert');

common.error('before');

// stack overflow
function stackOverflow() {
  stackOverflow();
}
stackOverflow();

common.error('after');