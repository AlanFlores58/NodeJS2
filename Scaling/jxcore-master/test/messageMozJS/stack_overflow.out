before
InternalError: too much recursion (*test*messageMozJS*stack_overflow.js 14:2)
    at stackOverflow (*test*messageMozJS*stack_overflow.js:14:3)
    at stackOverflow (*test*messageMozJS*stack_overflow.js:14:3)
    at stackOverflow (*test*messageMozJS*stack_overflow.js:14:3)
    at stackOverflow (*test*messageMozJS*stack_overflow.js:14:3)
    at stackOverflow (*test*messageMozJS*stack_overflow.js:14:3)
    at stackOverflow (*test*messageMozJS*stack_overflow.js:14:3)
    at stackOverflow (*test*messageMozJS*stack_overflow.js:14:3)
    at stackOverflow (*test*messageMozJS*stack_overflow.js:14:3)
    at stackOverflow (*test*messageMozJS*stack_overflow.js:
